python/cudf/cudf/pandas/__init__.py

```diff
@@ -25,7 +25,11 @@ def install():
     global LOADED
     LOADED = loader is not None
 
-    if (rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", None)) is not None:
+    if (
+        rmm_mode := os.getenv("CUDF_PANDAS_RMM_MODE", "managed_pool")
```

Comment:
os.getenv can’t return None, now that we provide a default value that is not None. Let’s refactor this now that the conditional and walrus operator are not needed.

Reply:
Done